float64[] Finger_pos
float64[] Finger_vel
float64[] Finger_acc
float64[] Finger_eff
duration time_from_start